website/docs/r/automation_hybrid_runbook_worker.html.markdown

```diff
@@ -0,0 +1,69 @@
+---
+subcategory: "Automation"
+layout: "azurerm"
+page_title: "Azure Resource Manager: azurerm_automation_hybrid_runbook_worker"
+description: |-
+  Manages a Automation.
+---
+
+# azurerm_automation_hybrid_runbook_worker
+
+Manages a Automation Hybrid Runbook Worker.
+
+## Example Usage
+
+```hcl
+resource "azurerm_automation_hybrid_runbook_worker" "example" {
+  resource_group_name     = azurerm_resource_group.test.name
+  automation_account_name = azurerm_automation_account.test.name
+  worker_group_name       = azurerm_automation_hybrid_runbook_worker_group.test.name
+  vm_resource_id          = azurerm_linux_virtual_machine.test.id
+  worker_id               = "00000000-0000-0000-0000-000000000000"
+}
+```
+
+## Arguments Reference
+
+The following arguments are supported:
+
+* `resource_group_name` - (Required) The name of the Resource Group where the Automation should exist. Changing this forces a new Automation to be created.
+
+* `automation_account_name` - (Required) The name of the automation account in which the Hybrid Worker is created. Changing this forces a new resource to be created.
+
+* `worker_group_name` - (Required) The name of the HybridWorker Group. Changing this forces a new Automation to be created.
+
+* `worker_id` - (Required) The ID of the HybridWorker. Changing this forces a new Automation to be created.
+
+* `vm_resource_id` - (Required) The ID of the virtual machine used for this HybridWorker. Changing this forces a new Automation to be created.
+
+## Attributes Reference
+
+In addition to the Arguments listed above - the following Attributes are exported: 
+
+* `id` - The ID of the Automation Hybrid Runbook Worker.
+
+* `ip` - The IP address of assigned machine .
+
+* `last_seen_date_time` - Last Heartbeat from the Worker.
+
+* `registered_date_time` - The registration time of the worker machine.
```

Comment:
wrong tense?
```suggestion
* `registration_date_time` - The registration time of the worker machine.
```

Reply:
en...the property name from the swagger definition is `registeredDateTime`

[hybridRunbookWorker.json#L377](https://github.com/Azure/azure-rest-api-specs/blob/56eb34160d11aeff73c20a81d2a485bc0966a26e/specification/automation/resource-manager/Microsoft.Automation/stable/2021-06-22/hybridRunbookWorker.json#L377)